### PR TITLE
TW-442 GitHub Actions integration

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
GitHub Actions CI integration topic.

- Created a new page for this CI integration.
- Page is accessible from the left navigation menu and the "CI Integrations" page at `src/pages/docs/integrations/ci-integrations.md`

### CODE REVIEW

**Review by jkonrath-postman — APPROVED** [review]
LGTM!

**Review by aditi-umashankar-pm — APPROVED** [review]

File: src/pages/docs/integrations/available-integrations/ci-integrations/github-actions.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+1. Select the **Operating system** for your CI/CD pipeline.
+1. Select **Copy Postman CLI Command** to copy the Postman CLI configuration.
+
+   <img alt="Generate the Postman CLI configuration" src="https://assets.postman.com/postman-docs/v10/generate-postman-cli-v10-b.jpg" width="548px">
+
+To add the Postman CLI configuration to your GitHub pipeline:
+
+1. Create a new YAML file in the `.github/workflows` directory in your GitHub repository, and then edit the file.
+1. Add the Postman CLI configuration you copied from Postman to the YAML file:
+    * Replace all instances of `$POSTMAN_API_KEY` with a valid [Postman API Key](/docs/developer/intro-api/#generating-a-postman-api-key).
```
Comment (shubhbhargav) [inline src/pages/docs/integrations/available-integrations/ci-integrations/github-actions.md:101]: @jkonrath-postman @MatthewPinsker-postman We should recommend using Secrets in Github for this. We have already updated the template for Github actions to incorporate the usage of secrets i.e. we are using `${{ secrets.POSTMAN_API_KEY }}` and the user can store their Postman API key in the secrets to keep it secure.
Committing the key would have security implications.

cc @malvikach

File: src/pages/docs/integrations/available-integrations/ci-integrations/github-actions.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+      - uses: actions/checkout@v3
+      - name: Install Postman CLI
+        run: |
+          curl https://dl-cli.pstmn.io/install/linux64 -o postman-cli.tar.gz
+          tar -zxvf postman-cli.tar.gz
+          sudo mv postman-cli /usr/bin/postman
+          rm postman-cli.tar.gz
+      # Runs a single command using the runners shell
+      - name: Login to Postman CLI
+        run: postman login --with-api-key $POSTMAN_API_KEY
```
Comment (shubhbhargav) [inline src/pages/docs/integrations/available-integrations/ci-integrations/github-actions.md:125]: This has been updated in the latest prod canary. Lets copy the entire template once from there.

**Review by aditi-umashankar-pm — APPROVED** [review]

File: src/pages/docs/integrations/available-integrations/ci-integrations/github-actions.md (hunk 1 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/integrations/available-integrations/ci-integrations/github-actions.md:102]: @shubhbhargav I checked the Postman CLI config on beta and I see the `$POSTMAN_API_KEY` variable. Should I expect to see the `${{ secrets.POSTMAN_API_KEY }}` variable instead?

In the UI, we explain that Postman recommends saving the key as an environment variable in the CI/CD pipeline. Would a similar mention be sufficient? Or do you think we need to specifically call out GitHub's Secrets feature?

File: src/pages/docs/integrations/available-integrations/ci-integrations/github-actions.md (hunk 2 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/integrations/available-integrations/ci-integrations/github-actions.md:127]: YAML file example is updated. Please see the latest commit.

File: src/pages/docs/integrations/available-integrations/ci-integrations/github-actions.md (hunk 1 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/integrations/available-integrations/ci-integrations/github-actions.md:102]: @shubhbhargav I added a note explaining the secrets recommendation. Thanks!

**Review by malvikach — APPROVED** [review]

**Pull request merged**